web/src/stream_create.js

```diff
@@ -298,6 +299,10 @@ function create_stream() {
                 xhr,
                 $(".stream_create_info"),
             );
+
+            // Subtract the height of `.stream_create_info` from the height of
+            // `.stream-creation-body` to prevent overlapping with `.modal-footer`.
+            $(".stream-creation-body").height("-=" + $(".stream_create_info").outerHeight(true));
```

Comment:
We should try hard to avoid this sort of solution -- layout should be done with CSS, not hacky JS computations.

Reply:
sure i will update the PR to do it with CSS only.